css-url-image-value-and-simple-length.html

```diff
@@ -0,0 +1,42 @@
+<script>
+
+var url = (function() {
+  var c = document.location.href.split('/');
+  c[c.length - 1] = 'resources/1x1-green.png';
+  return c.join('/');
+})();
+
+var fn = [
+  ["image with CSS OM", function() { element.style.backgroundImage = 'url(resources/1x1-green.png)'; }],
+  ["image with CSS Typed OM", function() { element.styleMap.set("background-image", new CSSURLImageValue(url)); }],
+  ["length with CSS OM", function() { element.style.width = "10px"; }],
+  ["length with CSS Typed OM", function() { element.styleMap.set("width", new CSSSimpleLength(10, 'px')); }]
+];
+
+var element = document.createElement('div');
+document.documentElement.append(element);
+
+var numOfOperations = [1, 10, 100, 1000, 10000, 100000];
+
+for (var op = 0; op < numOfOperations.length; ++op) {
```

Comment:
This set of quadruply-nested loops makes this hard to understand. It's fine to just have one test with a default number of operations instead of looping through different values here.